sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -130,7 +140,14 @@ public static ClientPipeline Create(
         int perCallIndex = index;
 
         // Add retry policy.
-        policies[index++] = options.RetryPolicy ?? ClientRetryPolicy.Default;
+        if (loggingOptions == null || loggingOptions.ShouldUseDefaultRetryPolicy())
```

Comment:
Why would we ask logging options which retry policy to use?

Reply:
Changed this